tests/volcano_test.go

```diff
@@ -0,0 +1,84 @@
+package tests
+
+import (
+	"fmt"
+	"github.com/mesosphere/kudo-spark-operator/tests/utils"
+	"github.com/stretchr/testify/suite"
+	"testing"
+)
+
+const volcanoInstallerUrl = "https://raw.githubusercontent.com/volcano-sh/volcano/release-0.2/installer/volcano-development.yaml"
```

Comment:
is it worth maintaining a copy of this spec in test resource folder instead of downloading it?

Reply:
yes, makes sense, switched to using local copy